core/src/main/java/com/gentics/mesh/cli/BootstrapInitializerImpl.java

```diff
@@ -250,6 +250,14 @@ private boolean initLocalData(MeshOptions configuration, boolean isJoiningCluste
 				// Only execute the changelog if there are any elements in the graph
 				invokeChangelog();
 			}
+
+			if (isJoiningCluster && requiresChangelog()) {
```

Comment:
How can you guarantee that the changes have been replicated from another instance?

Reply:
`db.clusterManager().start();` will block during activation of the server until the database has been synced.